labgrid/driver/modbusrtudriver.py

```diff
@@ -0,0 +1,64 @@
+from importlib import import_module
+import attr
+
+from ..factory import target_factory
+from .common import Driver
+
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class ModbusRTUDriver(Driver):
```

Comment:
What kind of device are you using this driver with? Rather than a blanked implementation where we basically pass through all the read/write functions, I'd rather add a specific device driver on top.

Reply:
I theory I agree. But I'm not sure which abstract driver it should implement. Modbus has a lot in common with the SerialDriver - how ever it does not make sense (IMO) to implement it and thereby give the user access to raw read() and write() methods, which would require the user to implement framing, CRC, ACK/NACK, timing, etc. that this driver is there to handle.

An other option would be to let this class inherit from something with eeprom as Modbus basically just allows you to read/write specific addresses on another device. However, I don't think any of the existing drivers offers that level of details.